### ISSUE TITLE
Ledger Nano S + Neon 0.1.2 Windows 10 Can't send GAS

### ISSUE THREAD

**Issue by GHuser357** [CityOfZion/neon-wallet#685]
When trying to send GAS, immediate "Error: Transaction Failed!", device screen remains "Wake Up, NEO..."

Tried every Neon version down to 0.7.0, on 2 computers, and reinstalling device app. No problem claiming GAS or Sending NEO. 

Edit: 19 Feb 18
I had never tried sending an amount of GAS less than one (1).  It seems I can send any amount <1, but any amount that contains a non zero integer fails.

Edit: 12 Mar 18
No change with 0.2.1

**Comment by ned-robinson** [comment]
I'm having the exact same issue. Claiming Gas or sending NEO works just fine. Can't send Gas though.

**Comment by KnowledgeSupreme** [comment]
Same issue.  using Neon version 0.1.3, and installed the latest firmware, and app version on the ledger wallet.  Can claim gas, but can't send gas. Get the error message, and the notification to confirm never shows up on the Ledger.

**Comment by ned-robinson** [comment]
So what do we do? I have a lot of GAS that can't be used. Is there any
other software that works with the Nano Ledger S?

Ned Robinson
Director of Software Development
eAlchemy
ned@eAlchemyLabs.com <chris@eAlchemyLabs.com>
415.361.7665

**Comment by JaimeCruzH** [comment]
**Neon 0.1.3** + Ledger Nano S + Windows 10 same problem. Can't send GAS

**Comment by brianlenz** [comment]
If you're still having issues sending GAS from your Ledger with Neon 0.1.4, make sure you are running the latest version 1.1.1 of the Ledger Neo app:

https://support.ledgerwallet.com/hc/en-us/articles/115005530425-How-to-install-and-use-Neo-NEO-

You need to delete the current Neo app from your Ledger (using the Ledger Manager Chrome app) and reinstall in order to complete the upgrade.

I know some of you have said you're on the latest firmware, so this may not solve it for all of you, but it's worth triple-checking. I've run some tests on my Ledger and sending GAS has worked fine across multiple wallets on both MainNet and TestNet, with values >=1 and <1.

**Comment by brianlenz** [comment]
By the way, I don't believe the Ledger Manager shows the currently installed version from your Ledger. It only shows the latest version, so if you aren't sure exactly which version you are using, it's best to delete the app and reinstall to be certain. We've seen this fix the issue for every one of our users who is having issues with transaction signing.

**Comment by GHuser357** [comment]
Had tried that several times already, no change but thanks for the suggestion.

**Comment by brianlenz** [comment]
Bummer, sorry to hear that @GHuser357. The Neon team is still working on improvements, so here's to hoping an upcoming version will be fruitful.

**Comment by KnowledgeSupreme** [comment]
Remedied the issue b going into the Neon settings and toggling between the different block explorers.  That eventually got it working as far as sending gas.

**Comment by mcurtis06** [comment]
I fixed the issue after reading GHuser357's updated Feb 19 comment in the first post sending an increment of less than 1 GAS. That transaction succeeded. And then after that, I could successfully send larger amounts of GAS in whole numbers.

**Comment by ned-robinson** [comment]
I've been able to fix the issue myself. Here's what I did:

•    From NEON wallet, go to Manage Neon settings
•    Exported Wallet Recovery File
•    Then immediately, load Wallet Recovery File (pick the one you just exported)
•    Opened Nano Ledger in NEON
o    Sent 0.000001 to non-ledger private wallet – Worked!
o    Sent 1.0 to non-ledger private wallet – Didn’t work
o    Sent 0.000009 to non-ledger private wallet – Worked!
o    Sent 0.999999 to non-ledger private wallet – Worked but only sent 0.9. The ending decimals weren’t sent, dropped from transaction.

Oberservations
1.    There is sometimes a noticeable delay between getting the Sign Transaction in the NEON wallet and the Sign Transaction coming up on the NANO Ledger. One time it took over 30 seconds. So the NEON wallet thought the transaction failed but then the message showed up on the NANO Ledger. I had to close down the NEON WALLET to get things back in sync. (Did not have to reboot).
2.    Decimals after the 2nd place are getting truncated and removed from the send. Most of the time, not 100% of the time. 
3.    Very inconsistent experience sending GAS.
4.    Not sure if exporting and then importing the wallet recovery file did anything but it was after doing it that I got some success. Maybe something was corrupted that the recovery file cleared out?

In any case, I can now send GAS from the NEON wallet.
